tests: subsys: fs: nffs_fs_api: fix nRF5x failures

nvlsianpu opened the pull request:
CONF_FILE was overwritten by boilerplate.cmake so not proper
settings were took for tests. NRF5 flash driver had changed
its name to one defined by DT (see #5773) - which causes
incompatibility witch previous settings.

This patch solves this problems.


Signed-off-by: Andrzej Puzdrowski <andrzej.puzdrowski@nordicsemi.no>

nvlsianpu committed:
tests: subsys: fs: nffs_fs_api: fix nRF5x failures

CONF_FILE was overwritten by boilerplate.cmake so not proper
settings were took for tests. NRF5 flash driver had changed
its name to one defined by DT (see #5773) - which causes
incompatibility witch previous settings.

This patch solves this problems.


Signed-off-by: Andrzej Puzdrowski <andrzej.puzdrowski@nordicsemi.no>

carlescufi commented:
@nvlsianpu Can we take this opportunity to clean up `prj.conf`? It has duplicated values and is there a reason we rename `CONFIG_SOC_FLASH_NRF5_DEV_NAME`?

nvlsianpu committed:
tests: subsys: fs: nffs_fs_api: cleanup configuration for nRF5x

Removed CONFIG_FLASH_PAGE_LAYOUT because it is already selected
by NFFS kconfig itself. Some other keywords were duplicated.

Signed-off-by: Andrzej Puzdrowski <andrzej.puzdrowski@nordicsemi.no>
